pkg/environment/EnvironmentService.go

```diff
@@ -14,14 +14,17 @@
  * limitations under the License.
  */
 
-package cluster
+package environment
```

Comment:
i think we should keep this at /pkg/cluster/environment instead of /pkg/cluster

Reply:
done